docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +112,95 @@ significant changes in each minor release.
 
-We've got :ref:`examples and instructions on creating changelog fragments <changelogs_how_to>` in the Community Guide.
+If there are major problems in RC2, the cycle begins again with another release candidate and repeats until the core maintainers agree that all major problems have been fixed.
 
 
-Release candidates
-^^^^^^^^^^^^^^^^^^^
+.. _development_and_stable_version_maintenance_workflow:
 
-Before a new release or version of Ansible or ``ansible-core`` can be
-done, it will typically go through a release candidate process.
+Development and stable version maintenance workflow
+----------------------------------------------------
 
-This provides the Ansible community the opportunity to test these releases and report
-bugs or issues they might come across.
+In between releases, the Ansible community develops and maintains ``ansible-core`` on GitHub_, with a workflow that looks like this:
 
-Ansible and ``ansible-core`` tag the first release candidate (``RC1``)
-which is usually scheduled to last five business days. The final release
-is done if no major bugs or issues are identified during this period.
+ * Developers add new features and bug fixes to the ``devel`` branch.
+ * Each new feature and each notable bug fix includes a changelog fragment describing the work.
+ * The development team backports bug fixes to one, two, or three stable branches, depending on the severity of the bug. They do not backport new features.
+ * Release engineers create a minor release for each maintained version every three weeks to ensure that the latest bug fixes are available to users.
+ * At the end of the development period, the release engineers impose a feature freeze and the work of creating a new release begins.
 
-If there are major problems with the first candidate, a second candidate will
-be tagged (``RC2``) once the necessary fixes have landed.
-This second candidate lasts for a shorter duration than the first.
-If no problems have been reported after two business days, the final release is
-done.
+We generally do not provide fixes for unmaintained releases of Ansible, however, there can sometimes be exceptions for critical issues.
 
-More release candidates can be tagged as required, so long as there are
-bugs that the Ansible  or ``ansible-core`` core maintainers consider
-should be fixed before the final release.
+.. _GitHub: https://github.com/ansible/ansible
 
-.. _release_freezing:
+.. _release_changelogs:
 
-Feature freeze
-^^^^^^^^^^^^^^^
+Changelogs
+^^^^^^^^^^
 
-While there is a pending release candidate, the focus of core developers and
-maintainers will on fixes towards the release candidate.
+We generate changelogs based on fragments. Here is the generated changelog for 2.9_ as an example. When creating new features or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
```

Comment:
```suggestion
We generate changelogs based on fragments. When creating new features for existing modules and plugins or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
```
1) IMO to put this reference https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment is enough. There are good examples
2) fragments are not needed for new modules / plugins.

Reply:
I think a link to a changelog is useful, but we don't need an extra sentence for it - I'll add a link to the word `changelogs` in the first sentence, and folks who are curious can use that.